src/main/java/uk/gov/hmcts/ccd/data/caseaccess/CaseUserRepository.java

```diff
@@ -43,4 +43,12 @@ public void revokeAccess(final Long caseId, final String userId) {
 
         return namedQuery.getResultList();
     }
+
+    public List<String> findCaseRolesUserHasForACase(final Long caseId, final String userId) {
```

Comment:
The method name would be more readable if shorter. Could it be changed to `findCaseRoles()`? I think the parameters provide enough indications as to the filters used by the query.

Reply:
👍